clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/timer/ScheduleWithPersistentTimeoutMetaDataCommand.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.clustering.ejb.infinispan.timer;
+
+import org.wildfly.clustering.ejb.timer.TimeoutMetaData;
+import org.wildfly.clustering.server.infinispan.scheduler.ScheduleWithPersistentMetaDataCommand;
+
+/**
+ * Schedule command using persistent meta data.
+ * @author Paul Ferraro
+ */
+public class ScheduleWithPersistentTimeoutMetaDataCommand<I> extends ScheduleWithPersistentMetaDataCommand<I, TimeoutMetaData> {
```

Comment:
BTW Is the visibility intentional? Because it looks like this should be package-private given the package-private constructor.

Reply:
Also `@param <I>` would always be welcome.